apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: -6.4rem;
+  right: 0;
+  color: #D32E6A;
+  font-size: 8rem;
+  cursor: pointer;
+}
+
+.main-column {
+  width: calc(67% - 10px);
+  margin: 5px;
+}
+
+.side-column {
+  width: calc(33% - 10px);
+  margin: 5px;
+
+  img {
+    max-width: 100%;
+  }
+}
+
 .reactions {
+  .reactions-modal {
+    display: none;
+  }
+
+  &.open {
+    .reactions-modal {
+      display: block;
+      z-index: 10000;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+
+      &:before {
+        position: fixed;
+        display: block;
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: white;
+        opacity: 0.93;
+      }
+    }
+
+    .reactions-container {
+      display: flex;
```

Comment:
Properties should be ordered position, top, left, display, width, max-width, margin, transform

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)